src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -3655,5 +3658,496 @@ private static ImmutableArray<Symbol> GetProperties(CSharpCompilation comp, stri
         {
             return comp.GetMember<NamedTypeSymbol>(typeName).GetMembers().WhereAsArray(m => m.Kind == SymbolKind.Property);
         }
+
+        [Fact]
+        public void BaseArguments_01()
+        {
+            var src = @"
+using System;
+
+class Base
+{
+    public Base(int X, int Y)
+    {
+        Console.WriteLine(X);
+        Console.WriteLine(Y);
+    }
+
+    public Base() {}
+}
+
+record C(int X, int Y) : Base(X, Y)
```

Comment:
>Base [](start = 25, length = 4)

Please test an argument list in the base with a `class` and `struct` declaration (rather than a `record` declaration).